recipes/libs/openssl.yaml

```diff
@@ -40,6 +84,10 @@ buildScript: |
 
     popd
 
+    if [[ "$SHARED_ONLY" == "true" ]] ; then
+        find install -iname "*.a" -exec rm -rf {} \;
+    fi
+
```

Comment:
Why is this necessary? The install class takes care of this already. Static libraries are only kept in -dev packages.

Reply:
for both, openssl and zlib, it isn't possible to just build shared libraries. static libraries will be built everytime, and shared libs are optional and additional by flag. but than "both" and "shared" are the same result.

btw. if i choose "both", which one will be used implicitly?